src/roll20.pyj

```diff
@@ -49,7 +49,8 @@ def escapeRoll20Macro(text):
 def genRoll(dice, modifiers={}):
     roll = "[[" + dice
     for m in modifiers:
-        mod = modifiers[m]
+        mod = str(modifiers[m])

```

Comment:
shouldn't be necessary to cast it to a string as it should always be sent as a string from the dndbeyond side